config/Version.public.xcconfig

```diff
@@ -1,2 +1,2 @@
-VERSION_LONG = 24.3.0.0
+VERSION_LONG = 24.3.0.3
```

Comment:
This big jump is due to a hiccup in the automation, fixed by https://github.com/wordpress-mobile/WordPress-iOS/pull/22685/commits/1e8e01b2bfcfd6efe02ea95e3702e769166293c5. Unfortunately, the failure was after CI already committed the version bump.